src/mainboard/protectli/vault_jsl/mainboard.c

```diff
@@ -129,6 +131,15 @@ void mainboard_silicon_init_params(FSP_S_CONFIG *params)
 		descriptor_patch_pcie8_lane(gpio_get(GPP_C6));
 	}
 
+	/*
+	* Enable L1 only for WiFi, L0s doesn't work reliably for Atheros QCA6174.
```

Comment:
The word order threw me off:
```suggestion
	* Enable only L1 for WiFi, L0s doesn't work reliably for Atheros QCA6174.
```
Also, formatting of the comment lacks a space.

Reply:
squashed to https://github.com/Dasharo/coreboot/pull/708/commits/418f2765fca132e59eda7a01b5674b3fdba6552d